### PR TITLE
Adding Examples of Unit Model Costing Documentation

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Fixes/Resolves:
Issue #1360 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/costing/thickener.rst [hunk 1]
```diff
@@ -9,7 +9,8 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
 .. csv-table::
    :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "parameter_name", "1", ":math:`\text{dimensionless}`"
+   "Capital cost A parameter:math:`^1`", ":math:`A`", "capital_a_parameter", "4729.8", ":math:`\text{USD_{2007}/ft}}`"
```
Comment (agarciadiego) [inline docs/technical_reference/costing/thickener.rst]: error showing in the doc

File: docs/technical_reference/costing/thickener.rst [hunk 2]
```diff
@@ -9,7 +9,8 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
+   "Capital cost B parameter:math:`^1`", ":math:`B`", "capital_b_parameter", "37068", ":math:`\text{USD_{2007}}`"
```
Comment (agarciadiego) [inline docs/technical_reference/costing/thickener.rst]: subscript and superscript not showing correctly

File: docs/technical_reference/costing/clarifier.rst [hunk 3]
```diff
@@ -9,7 +13,19 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
 .. csv-table::
    :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "parameter_name", "1", ":math:`\text{dimensionless}`"
+   "Circular clarifier"
+   "Capital cost A parameter:math:`^1`", ":math:`A_{circular}`", "construction_a_parameter", "-6e-4", ":math:`\text{USD_{2011}/}\text{ft}^4`"
+   "Capital cost B parameter:math:`^1`", ":math:`B_{circular}`", "construction_b_parameter", "98.952", ":math:`\text{USD_{2011}/}\text{ft}^2`"
+   "Capital cost C parameter:math:`^1`", ":math:`C_{circular}`", "construction_c_parameter", "191806", ":math:`\text{USD_{2011}}`"
```
Comment (agarciadiego) [inline docs/technical_reference/costing/clarifier.rst]: subscripts and superscripts not showing correctly

**Review by agarciadiego — COMMENTED** [review]
General comment: superscripts and subscripts are not showing correctly un USD and in the parameter description. Also couldn't find CSTR, CSTR injection in the docs

**Review by agarciadiego — APPROVED** [review]
LGTM

File: docs/technical_reference/costing/cstr.rst
Comment (savannahsakhai) [inline docs/technical_reference/costing/cstr.rst:1]: References
----------
Aim to include at least one reference in most cases, but delete this section if no references used for cost relationships/default values 

This section needs to be removed

**Review by savannahsakhai — APPROVED** [review]
LGTM, my only suggestion would be to delete the section in the CSTR page mentioned in my comment.

File: docs/technical_reference/costing/cstr.rst
Comment (MarcusHolly) [reply docs/technical_reference/costing/cstr.rst:1]: Thanks for catching this - I just forgot to add it.

**Pull request merged**